.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
Hi Encarta, you made a great effort on this project. However, ensure there are no linter errors in your project.